statemachine/src/main/java/com/amazonaws/stepfunctions/cloudformation/statemachine/Translator.java

```diff
@@ -1,42 +1,74 @@
 package com.amazonaws.stepfunctions.cloudformation.statemachine;
 
-import com.amazonaws.services.stepfunctions.model.CloudWatchLogsLogGroup;
-import com.amazonaws.services.stepfunctions.model.LogDestination;
+import com.amazonaws.services.stepfunctions.model.Tag;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class Translator {
 
-    public static com.amazonaws.services.stepfunctions.model.LoggingConfiguration getLoggingConfiguration(LoggingConfiguration input) {
-        com.amazonaws.services.stepfunctions.model.LoggingConfiguration result = new com.amazonaws.services.stepfunctions.model.LoggingConfiguration();
-        result.setLevel(input.getLevel());
-        result.setIncludeExecutionData(input.getIncludeExecutionData());
+    public static com.amazonaws.services.stepfunctions.model.LoggingConfiguration getLoggingConfiguration(LoggingConfiguration cfnLoggingConfiguration) {
```

Comment:
Do we need the full class path? Where is the other one defined and why are they different? Same comment for others

Reply:
Full class path is necessary as we are converting between Step Functions and CloudFormation models, which share a number of class names. For example `getLoggingConfiguration` converts between `com.amazonaws.services.stepfunctions.model.LoggingConfiguration` and `com.amazonaws.stepfunctions.cloudformation.statemachine.LoggingConfiguration`.